platform/ios/scripts/package.sh

```diff
@@ -207,17 +222,18 @@ SEM_VERSION=$( git describe --tags --match=ios-v*.*.* --abbrev=0 | sed 's/^ios-v
 SHORT_VERSION=${SEM_VERSION%-*}
 if [[ ${BUNDLE_RESOURCES} ]]; then
     cp -pv LICENSE.md "${OUTPUT}/static/${NAME}.framework"
-    cp -rv platform/ios/app/Settings.bundle "${OUTPUT}/static/${NAME}.framework"
+    cp -rv platform/ios/app/Settings.bundle ${STATIC_SETTINGS_DIRECTORY}
```

Comment:
More like:

- ifabric
  - pkg/static/Mapbox.framework (holds **everything**)

- ipackage* flavors
  - pkg/
    - Settings.bundle
    - static/
    - dynamic/
  - pkg/static/
    - Mapbox.framework
    - Mapbox.bundle (resources in here)
  - pkg/dynamic/
    - Mapbox.framework (resources in here)